src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -117,11 +117,14 @@ export default class FilterableMultiSelect extends React.Component {
   handleOnStateChange = changes => {
     const { type } = changes;
     switch (type) {
-      case Downshift.stateChangeTypes.keyDownArrowDown:
       case Downshift.stateChangeTypes.keyDownArrowUp:
       case Downshift.stateChangeTypes.itemMouseEnter:
         this.setState({ highlightedIndex: changes.highlightedIndex });
         break;
+      case Downshift.stateChangeTypes.keyDownArrowDown:
+        this.setState({ highlightedIndex: changes.highlightedIndex });
+        this.setState({ isOpen: true });
```

Comment:
Could these be in the same `setState` call?

Reply:
Good call